spec/spec_helper.rb

```diff
@@ -0,0 +1,9 @@
+RSpec.configure do |config|
+  config.expect_with :rspec do |expectations|
+     expectations.include_chain_clauses_in_custom_matcher_descriptions = true
```

Comment:
In some places there are 2 spaces, in others there are 4, like this one. We should try to keep it consistent.